src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -106,6 +110,10 @@
         connectionManager.getParams().setDefaultMaxConnectionsPerHost(20);
         connectionManager.getParams().setMaxTotalConnections(22);
     }
+    private static transient Cache<String, BitbucketTeam> cachedTeam = new Cache(6, TimeUnit.HOURS);
+    private static transient Cache<String, List<BitbucketCloudRepository>> cachedRepositories = new Cache(3, TimeUnit.HOURS);
+    private BitbucketRepository cachedRepository;
+    private String cachedDefaultBranch;
```

Comment:
if any field should be transient this would be the field... doesn't matter as the class isn't serializable... but for future-proofing if you remove transient from the static final fields then add it here